### PR TITLE
docs(readme): update readme

### PR DESCRIPTION
**Pull request by Tagir-A**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
-# Eslint config for logistics
\ No newline at end of file
+ - [airbnb-config](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb)
+ - [prettier integrated with eslint](https://prettier.io/docs/en/eslint.html#use-eslint-to-run-prettier)
+ - Our custom rules
+
+### How to use
+You need to install the config itself. 
+Then use the provided command to install all the peerDeps.
+1) `npm i -D @dh/eslint-config-logistics`
```
Comment (jarjan) [inline README.md]: Can be markdown ordered list
```
1. First
2. Second
3. ...
```

And adding yarn alternative will be good too 😉

File: README.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+2) `npx install-peerdeps --dev eslint-config-airbnb`
+3) Put `"extends": ["eslint-config-logistics"]` in your `.eslintrc`
+
+> Written with [StackEdit](https://stackedit.io/).
```
Comment (jarjan) [inline README.md]: Delete this one?

File: README.md (hunk 1 above)
Comment (Tagir-A) [reply README.md]: But it's already markdown ordered list? 🤔 
screen shot 2018-11-19 at 18 39 33

Comment (Tagir-A) [reply README.md]: As for yarn - check out this article: https://iamturns.com/yarn-vs-npm-2018/
@jarjan

Comment (jarjan) [reply README.md]: Oops, my bad about list.

Comment (vinicius33) [reply README.md]: Please, add yarn 😉 . 
Also, yarn is still as relevant as npm. Don't let a medium(or whatever) article be your north star in this aspect.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
-# Eslint config for logistics
\ No newline at end of file
+ - [airbnb-config](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb)
+ - [prettier integrated with eslint](https://prettier.io/docs/en/eslint.html#use-eslint-to-run-prettier)
+ - Our custom rules
+
+### How to use
+You need to install the config itself. 
+Then use the provided command to install all the peerDeps.
+1. `npm i -D @dh/eslint-config-logistics`
```
Comment (vinicius33) [inline README.md]:
```suggestion
1. `npm i -D @dh/eslint-config-logistics`
1. `yarn add -D @dh/eslint-config-logistics`
```

**Pull request merged**